pkg/ccl/sqlccl/partition_test.go

```diff
@@ -1084,12 +1080,30 @@ func TestInitialPartitioning(t *testing.T) {
 		2: tsArgs("n3"),
 	}}
 	tc := testcluster.StartTestCluster(t, 3, tcArgs)
-	defer tc.Stopper().Stop(context.Background())
+
 	sqlDB := sqlutils.MakeSQLRunner(tc.Conns[0])
 	sqlDB.Exec(t, `CREATE DATABASE data`)
 	sqlDB.Exec(t, `USE data`)
 
+	// Disabling store throttling vastly speeds up rebalancing.
+	sqlDB.Exec(t, `SET CLUSTER SETTING server.declined_reservation_timeout = '0s'`)
```

Comment:
Hm.. can you just override directly via the *Settings in the testserverargs? IIRC these gossiped updates sometimes flake since you can't wait for them to propagate.

Reply:
Overriding directly isn't a good idea as you can get that updated wiped out by Gossip updates. I think this is fine (or rather, I'll take my chances). Especially for this one it shouldn't cause a problem if it flakes? The test will take a little longer for the initial snapshots, but not for all of them.

Reply:
👍